app/helpers/api/ordapi/getOrderItems.js

```diff
@@ -0,0 +1,11 @@
+import { getData } from 'buying-catalogue-library';
+import { logger } from '../../../logger';
+import { getEndpoint } from '../../../endpoints';
+
+export const getOrderItems = async ({ orderId, catalogueItemType, accessToken }) => {
+  const endpoint = getEndpoint({ api: 'ordapi', endpointLocator: 'getOrderItems', options: { orderId, catalogueItemType } });

```

Comment:
I would do what we chatted about the other day and start moving away from using this and doing what we do in endpoints and in here so it can be tested too.

Reply:
Yeah OK. Will give that a go 👍